frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/tier1-b/win-guest-tools-mount.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { OperatingSystem } from '../../../integration-tests/tests/utils/constants/wizard';
+import { testName } from '../../support';
+import { K8S_KIND } from '../../utils/const';
+import { ProvisionSource } from '../../utils/const/provisionSource';
+import { resourceTitle, row } from '../../views/selector';
+import * as wizard from '../../views/selector-wizard';
+import { tab } from '../../views/tab';
+
+const WIN_GT = 'windows-guest-tools';
+
+const verifyWinGuestTools = () => {
+  cy.get(wizard.wizardNavLink)
+    .contains('Storage')
+    .should('not.be.disabled')
+    .click();
+  cy.get(row)
+    .contains(WIN_GT)
+    .should('exist');
+  cy.get(wizard.wizardNavLink)
+    .contains('General')
+    .click();
+  cy.get(wizard.mountWinGT).uncheck();
+  cy.get(wizard.wizardNavLink)
+    .contains('Storage')
+    .click();
+  cy.get(row)
+    .contains(WIN_GT)
+    .should('not.exist');
+  cy.get('#create-vm-wizard-reviewandcreate-btn').click();
+  cy.get('.kubevirt-create-vm-modal__review-tab__footer').then(($ftr) => {
+    if ($ftr.find(wizard.startOnCreation).length > 0) {
+      cy.get(wizard.startOnCreation).uncheck();
+    }
+  });
+  cy.get('#create-vm-wizard-submit-btn').click();
+  cy.byLegacyTestID('kubevirt-wizard-success-result').should('be.visible');
+  cy.get('button.pf-m-primary')
+    // .contains('details')
+    .click();
+  cy.get(resourceTitle)
+    .contains(testName)
+    .should('exist');
+  tab.navigateToDisk();
+  cy.get(row)
+    .contains(WIN_GT)
+    .should('not.exist');
+};
+
+describe('Windows guest tool mount', () => {
+  before(() => {
+    cy.Login();
+    cy.createProject(testName);
+  });
+
+  after(() => {
+    cy.deleteResource(K8S_KIND.VM, `vm-${testName}`, testName);
+    cy.deleteTestProject(testName);
+  });
+
+  it('ID(CNV-7491) Test Windows guest tool mount control for template creation', () => {
+    cy.visitVMTemplatesList();
+    cy.get('#item-create').click();
+    cy.get('#wizard-link').click();
+    cy.get(wizard.vmName)
+      .clear()
+      .type(`tmpl-${testName}`);
```

Comment:
This can be a const, and remove the template in the after block.

Reply:
done